### ISSUE TITLE
Files missing for CODE 2.1

### ISSUE THREAD

**Issue by ajdunevent** [CollaboraOnline/Docker-CODE#5]
The following errors are received after upgrade to CODE 2.1:
```
[ forkit ] FTL  Neither /opt/collaboraoffice5.1/program/libsofficeapp.so or /opt/collaboraoffice5.1/program/libmergedlo.so exist.| kit/Kit.cpp:1948
[ loolwsd ] FTL  Failed to fork child processes.| wsd/LOOLWSD.cpp:2405
```
Server info:
Arch Linux 
Kernel 4.10.13-1-ARCH
Docker 17.04.0

Please let me know if there is any other troubleshooting information I can provide.

**Comment by hugues-git** [comment]
in file scripts/start-libreoffice.sh on line 28
--o:lo_template_path=/opt/collaboraoffice5.1 should be --o:lo_template_path=/opt/collaboraoffice5.3

You can run this to fix it:
sudo docker exec CONTAINER_ID sed -i 's/collaboraoffice5.1/collaboraoffice5.3/' start-libreoffice.sh

**Comment by thebearon** [comment]
This was fixed with 0766bad04130d29106471d3bd939726be2769872.
